src/frontend/web_application/src/modules/message/models/Message.ts

```diff
@@ -7,32 +7,53 @@ export class Message {
   constructor(props = {}) {
     Object.assign(this, props);
   }
+
```

Comment:
Why those space ? Feels like LF/CRLF conversion ?

Reply:
probably prettier

Reply:
I remember, it's autofix eslint